### PR TITLE
Allow Custom ellipseMode

### PR DESCRIPTION
**Pull request by codypearce**
I came across a case where a client requested "ellipse" on the menu items on iOS. This simply adds the option for a custom 'ellipseMode'.

### CODE REVIEW

File: src/MenuItem.js [hunk 1]
```diff
@@ -22,12 +22,16 @@ function MenuItem({
...
+  ellipsizeMode,
   ...props
 }) {
   const touchableProps = Platform.select({
     android: { background: TouchableNativeFeedback.SelectableBackground() },
     default: {},
   });
+  
+  let ellipsize = Platform.OS === 'ios' ? 'clip' : 'tail';
+  if(ellipsizeMode) ellipsize = ellipsizeMode;
```
Comment (mxck) [inline src/MenuItem.js]:
```suggestion
  if (ellipsizeMode) {
    ellipsize = ellipsizeMode;
  } else {
    ellipsize = Platform.OS === 'ios' ? 'clip' : 'tail';
  }
```

File: src/MenuItem.js [hunk 2]
```diff
@@ -22,12 +22,16 @@ function MenuItem({
...
   textStyle,
+  ellipsizeMode,
   ...props
 }) {
   const touchableProps = Platform.select({
     android: { background: TouchableNativeFeedback.SelectableBackground() },
     default: {},
   });
+  
+  let ellipsize = Platform.OS === 'ios' ? 'clip' : 'tail';
```
Comment (mxck) [inline src/MenuItem.js]:
```suggestion
  let ellipsize;
```

Comment (mxck) [inline src/MenuItem.js]: You can move this to `MenuItem.defaultProps`.

File: src/MenuItem.js [hunk 3]
```diff
@@ -22,12 +22,16 @@ function MenuItem({
   onPress,
   style,
   textStyle,
+  ellipsizeMode,
```
Comment (mxck) [inline src/MenuItem.js]: You need add this prop to `README.md` file.

**Comment by codypearce** [comment]
@mxck Thanks! I made the changes you requested, let me know if anything else needs changing.

**Pull request merged**